src/BinaryParsers/ElfBinary/Dwarf/DwarfMemoryReader.cs

```diff
@@ -121,7 +121,6 @@ public ulong ReadLength(out bool is64bit)
         /// <summary>
         /// Reads the string from the current position in the stream.
         /// </summary>
-        [HandleProcessCorruptedStateExceptions]
```

Comment:
I see this is no longer supported in .NET 8, but what are the implications of removing this attribute?